packages/file_selector/file_selector_linux/example/pubspec.yaml

```diff
@@ -9,7 +9,8 @@ environment:
 dependencies:
   file_selector_linux:
     path: ../
-  file_selector_platform_interface: ^2.2.0
+  file_selector_platform_interface:
+    path: ../../file_selector_platform_interface/
```

Comment:
This change shouldn't be committed

Reply:
I know, those will be removed after publishing the new version of platform_interface